orchestrator.Jenkinsfile

```diff
@@ -50,6 +50,7 @@ projects = [
     "tests/kubeless-test-client": "kubeless-test-client",
     "tests/api-controller-acceptance-tests": "api-controller-acceptance-tests",
     "tests/connector-service-tests": "connector-service-tests",
+    "tests/metadata-service-tests": "metadata-service-tests",
```

Comment:
This component will fail to be fetched from the docker registry because the Jenkinsfile for this does not push a docker image called `metadata-service-tests`.